src/Winium.Desktop.Driver/Winium.Desktop.Driver.csproj

```diff
@@ -116,6 +116,7 @@
     <Compile Include="CommandExecutors\NotImplementedExecutor.cs" />
     <Compile Include="CommandExecutors\SwitchToWindowExecutor.cs" />
     <Compile Include="CommandLineOptions.cs" />
+    <Compile Include="CommonHelpers\CommonHelpers.cs" />
```

Comment:
1. There is already [CommandHelpers](https://github.com/PersonifyInc/Winium.Desktop/pull/5/files#diff-28106b624ee73a148b179709813e78ccR134) we don't need to have CommonHelpers --> let's use what exists.
2. CommonHelpers is too common and general, look at the author is doing, each helper must have its meaning (e.g. [OSInfo.cs](https://github.com/PersonifyInc/Winium.Desktop/pull/5/files#diff-28106b624ee73a148b179709813e78ccR133) or [BuildInfo.cs](https://github.com/PersonifyInc/Winium.Desktop/pull/5/files#diff-28106b624ee73a148b179709813e78ccR134) ==> Let's use something like `TerminateApp.cs`